### PR TITLE
Functional tests - Renaming functions in files.js

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Renaming functions in files.js
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | no tests needed

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/18775)

### CODE REVIEW

File: tests/puppeteer/campaigns/functional/BO/02_orders/01_orders/03_exportOrders.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ describe('Export orders', async () => {
   it('should export orders to a csv file', async function () {
     await testContext.addContextItem(this, 'testIdentifier', 'exportOrders', baseContext);
     await this.pageObjects.ordersPage.exportDataToCsv();
-    const fileExist = await files.checkFileExistence('order_', 5000, true, 'csv');
+    const fileExist = await files.fileExist('order_', 5000, true, 'csv');
```
Comment (nesrineabdmouleh) [inline tests/puppeteer/campaigns/functional/BO/02_orders/01_orders/03_exportOrders.js]:
```suggestion
    const fileExist = await files.isFileExist('order_', 5000, true, 'csv');
```

Comment (SimonGrn) [reply tests/puppeteer/campaigns/functional/BO/02_orders/01_orders/03_exportOrders.js]:
```suggestion
    const fileExist = await files.doesFileExist('order_', 5000, true, 'csv');
```
Maybe ?

Comment (boubkerbribri) [reply tests/puppeteer/campaigns/functional/BO/02_orders/01_orders/03_exportOrders.js]: done thanks

**Review by SimonGrn — DISMISSED** [review]

File: tests/puppeteer/campaigns/functional/BO/02_orders/03_creditSlips/02_generateCreditSlipsByDate.js [hunk 2]
```diff
@@ -184,9 +184,9 @@ describe('Generate Credit slip file by date', async () => {
     });
 
     it('should generate PDF file by date and check the file existence', async function () {
-      await testContext.addContextItem(this, 'testIdentifier', 'generatePdfAndCheckFileExistence', baseContext);
+      await testContext.addContextItem(this, 'testIdentifier', 'generatePdfAnddoesFileExist', baseContext);
```
Comment (Progi1984) [inline tests/puppeteer/campaigns/functional/BO/02_orders/03_creditSlips/02_generateCreditSlipsByDate.js]:
```suggestion
      await testContext.addContextItem(this, 'testIdentifier', 'generatePdfAndDoesFileExist', baseContext);
```

File: tests/puppeteer/campaigns/utils/files.js [hunk 3]
```diff
@@ -19,7 +19,7 @@ module.exports = {
    * @param fileExtension
    * @return boolean, true if exist, false if not
    */
-  async checkFileExistence(fileName, timeDelay = 5000, isPartialName = false, fileExtension = '') {
+  async doesdoesFileExist(fileName, timeDelay = 5000, isPartialName = false, fileExtension = '') {
```
Comment (Progi1984) [inline tests/puppeteer/campaigns/utils/files.js]:
```suggestion
  async doesFileExist(fileName, timeDelay = 5000, isPartialName = false, fileExtension = '') {
```

Comment (boubkerbribri) [reply tests/puppeteer/campaigns/utils/files.js]: done thanks

**Review by Progi1984 — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Comment by matks** [comment]
Thank you @boubkerbribri

**Pull request merged**